tests/test_project.py

```
@@ -126,12 +126,12 @@ def test_name_set_from_set_design():
 
 
 def test_design_not_set():
-    with pytest.raises(ValueError, match="^design name is not set$"):
+    with pytest.raises(ValueError, match=r"^design name is not set$"):
         Project().design
```

Ruff (0.14.13)

Found useless attribute access. Either assign it to a variable or remove it.

(B018)